doc/src/modules/physics/mechanics/symsystem.rst

```diff
@@ -0,0 +1,184 @@
+=====================================
+Symbolic Systems in Physics/Mechanics
+=====================================
+
+The `SymbolicSystem` class in physics/mechanics is a location for the pertinent
+of a dynamic system. In its most basic form it contains the equations of motion
+for the dynamic system, however, it can also contain information regarding the
+loads that the system is feeling, the bodies that the system is comprised of
+and any additional equations the user feels is important for the system. The
+goal of this class is to provide a unified output format for the equations of
+motion that simulation code can be designed around.
+
+SymbolicSystem Example Usage
+============================
+
+This code will go over the manual input of the equations of motion for the
+simple pendulum into `SymbolicSystem` using x and y coordinates instead of
+theta.
+
+The equations of motion are formed at
+http://nbviewer.jupyter.org/github/bmcage/odes/blob/master/docs/ipython/Planar%20Pendulum%20as%20DAE.ipynb ::
+
+    >>> from sympy import atan, symbols, Matrix
+    >>> from sympy.physics.mechanics import (dynamicsymbols, ReferenceFrame,
+    ...                                      Particle, Point)
+    >>> import sympy.physics.mechanics.system as system
+
+The first step will be to initialize all of the dynamic and constant symbols. ::
+
+    >>> x, y, u, v, lam = dynamicsymbols('x y u v lambda')
+    >>> m, l, g = symbols('m l g')
+
+Next step is to define the equations of motion in multiple forms:
+
+    [1] Explicit form where the kinematics and dynamics are combined
+        x' = F(x, t, r, p)
```

Comment:
@moorepants I agree with Angadh that this would be a good change. It is worth noting I pulled this section from the code generator in pydy and so the change in notation is equally valid there.